package.json

```diff
@@ -33,10 +33,11 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "pre-commit": "yarn format && yarn lint && yarn ts && yarn test && yarn build",
-    "playwright:run": "yarn run playwright test playwright/e2e",
-    "playwright:debug": "yarn run playwright test playwright/e2e --debug",
-    "playwright:demo": "SLOW_MO=750 TIMEOUT=600000 yarn run playwright test playwright/e2e --headed",
-    "test:e2e": "yarn run wait-on -t 60 http://localhost:3001 && NODE_OPTIONS=--max-old-space-size=16384 yarn run playwright:run",
+    "playwright:run": "yarn run playwright test playwright/e2e --project=dev",
+    "playwright:debug": "yarn run playwright test playwright/e2e --debug --project=dev",
+    "playwright:demo": "SLOW_MO=750 TIMEOUT=600000 yarn run playwright test playwright/e2e --headed --project=dev",
+    "playwright:run:prod": "yarn run playwright test playwright/e2e --project=prod",
```

Comment:
Is there any way to make it default to a project of dev so that it doesn't have to be specified in most of these, and then can also be overridden if we want?

Reply:
The default is to run all projects. I couldn't find a way to disable that behavior:  
  
https://playwright.dev/docs/test-cli#run-tests  
  
The only option I could see would be to hide the prod tests behind an environment variable. Worth doing?

Reply:
Nah I think we're good!